### ISSUE TITLE
503 on all client endpoints aside from /actuator and /actuator/health

### ISSUE THREAD

**Issue by Na3blis** [codecentric/spring-boot-admin#1667]
I'm trying to use Spring Boot Admin in a GKE environment. I'm able to register the client to the server by using the ip=true property, however, the only thing that works is the health endpoint. When I look at the logs, it shows the server trying to access the `/actuator` and `/actuator/health` endpoints with `:8080` port, but the other endpoints, it does not have any port, and is resulting in a 503 when trying to retrieve any of the other information.

```
Couldn't retrieve info for Instance(id=514dbd0e8d5e, version=2, registration=Registration(name=spring-boot-application, managementUrl=http://x.x.x.x:8080/actuator, healthUrl=http://x.x.x.x:8080/actuator/health, serviceUrl=http://x.x.x.x:8080/, source=http-api), registered=true, statusInfo=StatusInfo(status=UP, details={livenessState={status=UP}, readinessState={status=UP}, db={status=UP, details={database=PostgreSQL, validationQuery=isValid()}}, diskSpace={status=UP, details={total=48393486336, free=43934580736, threshold=10485760, exists=true}}, ping={status=UP}}), statusTimestamp=2021-03-18T18:16:32.589044Z, info=Info(values={}), endpoints=Endpoints(endpoints={loggers=Endpoint(id=loggers, url=http://x.x.x.x/actuator/loggers), beans=Endpoint(id=beans, url=http://x.x.x.x/actuator/beans), health=Endpoint(id=health, url=http://x.x.x.x:8080/actuator/health), metrics=Endpoint(id=metrics, url=http://x.x.x.x/actuator/metrics), info=Endpoint(id=info, url=http://x.x.x.x/actuator/info)}), buildVersion=null, tags=Tags(values={})): 503 SERVICE_UNAVAILABLE
```

On the client service I have tho following in my application.yaml:
```
  boot:
    admin:
      client:
        url: http://spring-boot-admin-v1:8080
        instance:
          prefer-ip: true

management:
  endpoint:
    health:
      probes:
        enabled: true        # Force probe groups even when not running in k8s
      show-details: ALWAYS   # Include details
      group:
        liveness:            # liveness will restart the container but will not turn off routing,
          include: "diskSpace,ping,livenessStateProbeIndicator"
        readiness:           # readiness will stop routing
          include: "*"
  endpoints:
    web:
      exposure:
        include: "beans,loggers,health,info,metrics"
```

**Comment by SteKoe** [comment]
Hi @Na3blis,

could you please provide information about the used Spring Boot and Spring Boot Admin versions?

**Comment by Na3blis** [comment]
Hey @SteKoe we are currently using Spring Boot  2.4.5 and Spring Boot Admin Starter Client 2.3.1 for the services, and the server is running Spring Boot Admin Starter Server 2.4.1. I haven't tried the client 2.4.1 yet, but I'll be trying that later today

**Comment by ulischulte** [comment]
Since the problem seems to be solved (also see #1771), I'm closing this issue.
